2018/09/30 16:52:19 [error] 31306#0: *28 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 16:52:19 [error] 31307#0: *33 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 16:52:19 [error] 31307#0: *34 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:11:07 [error] 31307#0: *35 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:11:07 [error] 31306#0: *36 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:11:07 [error] 31307#0: *38 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:13:30 [error] 31306#0: *40 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:13:30 [error] 31307#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:13:30 [error] 31306#0: *43 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:30:03 [error] 31307#0: *44 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:30:03 [error] 31307#0: *45 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 17:30:03 [error] 31307#0: *47 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 18:50:02 [error] 31306#0: *73 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:50:02 [error] 31306#0: *72 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:13 [error] 31306#0: *82 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:13 [error] 31307#0: *78 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:15 [error] 31306#0: *81 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:15 [error] 31307#0: *78 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:16 [error] 31306#0: *82 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:16 [error] 31306#0: *81 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:16 [error] 31306#0: *82 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:51:16 [error] 31306#0: *81 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:54:02 [error] 31307#0: *96 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:54:06 [error] 31307#0: *96 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:54:21 [error] 31307#0: *96 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 18:54:51 [error] 31306#0: *104 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:07:41 [error] 31306#0: *108 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:07:41 [error] 31306#0: *108 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:07:42 [error] 31306#0: *108 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:07:42 [error] 31306#0: *107 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:08:04 [error] 31307#0: *116 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:34:23 [error] 31307#0: *120 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:35:03 [error] 31307#0: *120 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:35:04 [error] 31307#0: *120 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:35:10 [error] 31307#0: *120 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:35:10 [error] 31306#0: *119 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:35:34 [error] 31307#0: *120 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:35:48 [error] 31307#0: *120 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:48:00 [error] 31306#0: *133 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:48:34 [error] 31306#0: *140 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:49:48 [error] 31306#0: *142 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:02 [error] 48656#0: *2 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:27 [error] 48657#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:29 [error] 48657#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:29 [error] 48657#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:29 [error] 48657#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:29 [error] 48656#0: *2 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:30 [error] 48657#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:50:39 [error] 48657#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:49 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:50 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:50 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:50 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:51 [error] 48656#0: *17 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:51 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:51 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:51 [error] 48656#0: *17 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:51 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:51 [error] 48656#0: *17 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:52:52 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:53:27 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:53:28 [error] 48656#0: *18 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:54:44 [error] 48656#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:54:45 [error] 48656#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:54:45 [error] 48656#0: *40 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:54:45 [error] 48656#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:54:45 [error] 48656#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:54:45 [error] 48656#0: *40 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:55:02 [error] 48656#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:57:05 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:57:07 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:08 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:50 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:51 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:51 [error] 48656#0: *52 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:51 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:51 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:51 [error] 48656#0: *52 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:51 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:58:59 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:59:00 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:59:43 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 19:59:55 [error] 48656#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:14:54 [error] 48656#0: *88 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:14:54 [error] 48656#0: *80 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:15:50 [error] 48657#0: *3726 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/editor.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:15:50 [error] 48657#0: *3706 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/index.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:15:51 [error] 48656#0: *3713 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/markdown.md HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:15:54 [error] 48657#0: *3896 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:15:54 [error] 48657#0: *3706 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:16:52 [error] 48657#0: *7593 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/markdown.md HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:16:52 [error] 48656#0: *7897 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/index.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:16:53 [error] 48656#0: *7761 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/editor.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:16:55 [error] 48656#0: *7907 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:16:55 [error] 48656#0: *7761 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:19:25 [error] 48657#0: *21002 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/index.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:19:26 [error] 48656#0: *21046 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/editor.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:19:26 [error] 48656#0: *20887 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/markdown.md HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:19:33 [error] 48657#0: *21098 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:19:33 [error] 48657#0: *21002 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:02 [error] 48657#0: *23418 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:02 [error] 48656#0: *23417 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:02 [error] 48656#0: *23471 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:02 [error] 48657#0: *23418 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:03 [error] 48657#0: *23527 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:03 [error] 48656#0: *23242 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:04 [error] 48657#0: *23608 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:04 [error] 48656#0: *23470 open() "/opt/yxf_mysite_py_django/mysite/static/lib/jquery/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/lib/jquery/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:08 [error] 48657#0: *23608 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/editor.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:08 [error] 48657#0: *23607 upstream prematurely closed connection while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/index.css HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:20:09 [error] 48657#0: *23605 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.2.1, server: localhost, request: "GET /blog/markdown.md HTTP/1.1", upstream: "uwsgi://127.0.0.1:9090", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:30:08 [error] 48656#0: *55998 open() "/opt/yxf_mysite_py_django/mysite/static/plugins/JQuery/jquery-3.3.1.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/plugins/JQuery/jquery-3.3.1.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:31:35 [error] 48657#0: *56002 open() "/opt/yxf_mysite_py_django/mysite/static/plugins/JQuery/jquery-3.3.1.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/plugins/JQuery/jquery-3.3.1.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/blog/"
2018/09/30 20:31:53 [error] 48657#0: *56002 open() "/opt/yxf_mysite_py_django/mysite/static/plugins/JQuery/jquery-3.3.1.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/plugins/JQuery/jquery-3.3.1.min.js HTTP/1.1", host: "192.168.2.135"
2018/09/30 20:38:58 [error] 48656#0: *56019 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:38:58 [error] 48657#0: *56021 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:38:58 [error] 48657#0: *56022 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:38:58 [error] 48656#0: *56019 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:38:58 [error] 48656#0: *56024 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:38:58 [error] 48657#0: *56025 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:39:01 [error] 48656#0: *56019 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:39:01 [error] 48657#0: *56022 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:39:01 [error] 48657#0: *56021 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:39:01 [error] 48656#0: *56024 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:39:01 [error] 48656#0: *56019 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:44:13 [error] 48656#0: *56039 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48657#0: *56046 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48656#0: *56048 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48656#0: *56049 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48656#0: *56050 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48657#0: *56047 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48657#0: *56047 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:44:13 [error] 48657#0: *56047 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/09/30 20:45:33 [error] 48656#0: *56054 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:33 [error] 48657#0: *56057 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:33 [error] 48657#0: *56058 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:33 [error] 48657#0: *56056 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:33 [error] 48657#0: *56058 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:33 [error] 48656#0: *56054 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:34 [error] 48656#0: *56054 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:34 [error] 48656#0: *56054 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:34 [error] 48657#0: *56057 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:34 [error] 48657#0: *56058 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/09/30 20:45:34 [error] 48656#0: *56054 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: ::1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2018/10/01 00:46:23 [error] 54138#0: *27 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *29 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *30 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *31 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *32 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *33 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *33 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 00:46:23 [error] 54137#0: *33 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:11:27 [error] 54137#0: *37 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *40 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *42 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *43 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *40 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54137#0: *39 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *44 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *44 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:06 [error] 54138#0: *44 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 01:13:20 [error] 54138#0: *44 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1556#0: *1 open() "/opt/yxf_mysite_py_django/mysite/static/css/bootstrap.min.css" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/css/bootstrap.min.css HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1553#0: *2 open() "/opt/yxf_mysite_py_django/mysite/static/images/begin.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/begin.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1556#0: *4 open() "/opt/yxf_mysite_py_django/mysite/static/images/online-try.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/online-try.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1553#0: *6 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1556#0: *5 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1553#0: *7 open() "/opt/yxf_mysite_py_django/mysite/static/images/references.jpg" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/references.jpg HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1553#0: *7 open() "/opt/yxf_mysite_py_django/mysite/static/js/jquery.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/jquery.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:32:44 [error] 1553#0: *7 open() "/opt/yxf_mysite_py_django/mysite/static/js/bootstrap.min.js" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/js/bootstrap.min.js HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:50:33 [error] 1553#0: *26 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:50:49 [error] 1556#0: *36 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135"
2018/10/01 08:52:03 [error] 1556#0: *40 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:52:16 [error] 1553#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:52:16 [error] 1553#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:52:17 [error] 1553#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 08:52:17 [error] 1553#0: *41 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:04:03 [error] 1553#0: *51 open() "/opt/yxf_mysite_py_django/mysite/static/images/newlogo.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/newlogo.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:04:03 [error] 1553#0: *52 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:08:34 [error] 1553#0: *53 open() "/opt/yxf_mysite_py_django/mysite/static/images/newlogo.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/newlogo.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:08:34 [error] 1553#0: *54 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:10:44 [error] 1553#0: *57 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:10:51 [error] 1553#0: *57 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:10:52 [error] 1553#0: *57 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:18:10 [error] 1553#0: *63 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:20:58 [error] 1556#0: *66 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:20:59 [error] 1556#0: *66 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:21:00 [error] 1556#0: *66 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:22:47 [error] 1556#0: *71 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:22:50 [error] 1556#0: *71 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:23:33 [error] 1556#0: *71 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:20 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:20 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:36 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:37 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:37 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:37 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:25:37 [error] 1556#0: *77 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:30:35 [error] 1556#0: *87 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 09:30:39 [error] 1556#0: *87 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:30:40 [error] 1556#0: *87 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:31:53 [error] 1553#0: *99 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:31:54 [error] 1556#0: *95 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:32:06 [error] 1553#0: *98 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:32:48 [error] 1553#0: *99 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:33:02 [error] 1556#0: *106 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:41:02 [error] 1556#0: *107 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:41:14 [error] 1556#0: *112 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:41:24 [error] 1556#0: *112 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:46:20 [error] 1553#0: *116 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:47:00 [error] 1553#0: *116 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:47:01 [error] 1553#0: *116 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:48:31 [error] 1553#0: *121 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:48:34 [error] 1556#0: *126 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "hcmw-yym.imwork.net", referrer: "http://hcmw-yym.imwork.net/"
2018/10/01 09:51:46 [error] 1556#0: *127 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:52:54 [error] 1556#0: *130 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 09:53:50 [error] 1556#0: *130 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:01:19 [error] 1556#0: *135 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:05:24 [error] 1553#0: *138 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:08:08 [error] 1556#0: *144 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:11:35 [error] 1556#0: *145 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:12:51 [error] 1556#0: *148 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:12:51 [error] 1556#0: *148 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:18:04 [error] 1553#0: *151 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:18:04 [error] 1553#0: *151 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:18:31 [error] 1553#0: *151 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:18:31 [error] 1553#0: *151 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:20:11 [error] 1556#0: *156 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:20:11 [error] 1556#0: *156 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:22:57 [error] 1556#0: *159 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:22:57 [error] 1556#0: *159 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:24:45 [error] 1553#0: *162 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:24:45 [error] 1553#0: *162 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:25:59 [error] 1553#0: *165 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:25:59 [error] 1553#0: *165 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:26:01 [error] 1553#0: *165 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:26:01 [error] 1553#0: *165 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:26:01 [error] 1553#0: *165 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:26:01 [error] 1553#0: *165 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:30:43 [error] 1553#0: *171 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:30:43 [error] 1553#0: *171 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:32:36 [error] 1556#0: *175 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:32:37 [error] 1556#0: *175 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:33:55 [error] 1556#0: *177 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:33:55 [error] 1553#0: *176 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:34:59 [error] 1556#0: *177 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:34:59 [error] 1556#0: *177 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/static/css/basic.css"
2018/10/01 10:38:46 [error] 1556#0: *181 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:38:46 [error] 1556#0: *181 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/static/css/basic.css"
2018/10/01 10:40:09 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:40:09 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/static/css/basic.css"
2018/10/01 10:40:10 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:40:10 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/static/css/basic.css"
2018/10/01 10:40:19 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:40:19 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/img/explorer.png" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/img/explorer.png HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/static/css/basic.css"
2018/10/01 10:40:20 [error] 1556#0: *184 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:43:39 [error] 1556#0: *190 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:44:43 [error] 1556#0: *189 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:52:40 [error] 1553#0: *196 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 10:52:47 [error] 1553#0: *196 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:01:08 [error] 1553#0: *198 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:04:11 [error] 1556#0: *203 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:06:07 [error] 1556#0: *204 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:06:45 [error] 1553#0: *208 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:06:58 [error] 1556#0: *204 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:08:12 [error] 1553#0: *210 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:08:25 [error] 1553#0: *210 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:08:26 [error] 1553#0: *210 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:08:40 [error] 1553#0: *210 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:10:18 [error] 1556#0: *220 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:12:56 [error] 1556#0: *221 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:13:11 [error] 1556#0: *225 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:15:38 [error] 1556#0: *226 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:16:11 [error] 1553#0: *229 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:18:40 [error] 1556#0: *231 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:21:26 [error] 1553#0: *236 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:22:37 [error] 1553#0: *237 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:22:58 [error] 1553#0: *237 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:28:04 [error] 1556#0: *243 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:28:34 [error] 1556#0: *243 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:30:00 [error] 1556#0: *247 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:30:00 [error] 1556#0: *247 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:30:01 [error] 1556#0: *247 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:30:39 [error] 1556#0: *247 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:31:23 [error] 1553#0: *257 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:36:29 [error] 1553#0: *259 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:37:27 [error] 1553#0: *258 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:43:11 [error] 1553#0: *265 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:43:29 [error] 1556#0: *263 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:43:49 [error] 1553#0: *265 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:44:12 [error] 1556#0: *263 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:44:28 [error] 1553#0: *265 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 11:44:46 [error] 1556#0: *263 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:03:12 [error] 1556#0: *272 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:03:49 [error] 1556#0: *271 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:03:51 [error] 1556#0: *272 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:03:51 [error] 1556#0: *271 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:03:52 [error] 1556#0: *272 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:04:17 [error] 1556#0: *271 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:04:32 [error] 1556#0: *272 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:06:03 [error] 1556#0: *282 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:06:03 [error] 1556#0: *282 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:06:04 [error] 1556#0: *282 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:06:04 [error] 1556#0: *282 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:07:07 [error] 1553#0: *280 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:07:08 [error] 1553#0: *280 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/?"
2018/10/01 12:07:14 [error] 1553#0: *280 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:34 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:35 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:35 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:35 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:36 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:36 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:36 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:37 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:37 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:38 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:39 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:47 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:48 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:48 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:49 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:10:59 [error] 1553#0: *309 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:00 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:00 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:00 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:01 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:01 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:26 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:27 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:27 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:27 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:28 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:11:28 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:12:02 [error] 1553#0: *307 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:12:03 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:12:03 [error] 1553#0: *291 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:12:08 [error] 1553#0: *307 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:14:50 [error] 1553#0: *326 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:15:10 [error] 1553#0: *325 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:15:32 [error] 1553#0: *326 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:16:54 [error] 1556#0: *332 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:17:11 [error] 1556#0: *330 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:17:56 [error] 1556#0: *332 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:18:30 [error] 1556#0: *330 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:18:39 [error] 1556#0: *332 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:18:57 [error] 1556#0: *330 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:21:29 [error] 1553#0: *340 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:21:51 [error] 1553#0: *338 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:23:19 [error] 1553#0: *343 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:49:44 [error] 1553#0: *346 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:50:12 [error] 1556#0: *345 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:50:31 [error] 1556#0: *345 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:54:08 [error] 1553#0: *351 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:54:13 [error] 1553#0: *350 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:54:14 [error] 1553#0: *350 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:54:14 [error] 1553#0: *351 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:55:16 [error] 1553#0: *350 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:55:20 [error] 1553#0: *351 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:55:20 [error] 1553#0: *351 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:55:21 [error] 1553#0: *350 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 12:56:57 [error] 1556#0: *361 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:02:39 [error] 1556#0: *364 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:08:57 [error] 1556#0: *367 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:10:08 [error] 1556#0: *367 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:10:52 [error] 1556#0: *387 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:13:08 [error] 1556#0: *401 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:13:39 [error] 1556#0: *401 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:24:30 [error] 1556#0: *434 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:24:58 [error] 1553#0: *430 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:29:43 [error] 1556#0: *437 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:30:06 [error] 1553#0: *436 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:31:09 [error] 1556#0: *437 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:32:02 [error] 1553#0: *436 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:33:22 [error] 1556#0: *445 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:33:44 [error] 1556#0: *444 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:34:07 [error] 1556#0: *445 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:34:21 [error] 1556#0: *444 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:34:40 [error] 1556#0: *445 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:34:51 [error] 1556#0: *444 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:35:07 [error] 1556#0: *445 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:36:55 [error] 1553#0: *454 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:37:05 [error] 1553#0: *453 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:37:12 [error] 1553#0: *454 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:37:41 [error] 1553#0: *453 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:38:10 [error] 1553#0: *454 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:38:21 [error] 1553#0: *453 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:38:39 [error] 1553#0: *454 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:38:56 [error] 1553#0: *453 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:39:15 [error] 1553#0: *454 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:39:26 [error] 1553#0: *453 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:42:06 [error] 1556#0: *466 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:42:37 [error] 1556#0: *465 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:44:08 [error] 1556#0: *470 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:46:15 [error] 1553#0: *473 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:46:31 [error] 1556#0: *472 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:47:53 [error] 1553#0: *478 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:50:16 [error] 1553#0: *481 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:50:40 [error] 1556#0: *479 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:51:27 [error] 1553#0: *481 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:51:58 [error] 1556#0: *479 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:52:29 [error] 1556#0: *479 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:52:53 [error] 1556#0: *479 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:53:15 [error] 1553#0: *481 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:53:46 [error] 1553#0: *481 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:53:58 [error] 1553#0: *481 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:55:07 [error] 1556#0: *492 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:55:30 [error] 1553#0: *481 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:55:46 [error] 1556#0: *492 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:58:52 [error] 1553#0: *497 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:59:07 [error] 1556#0: *496 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 13:59:27 [error] 1553#0: *497 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:20:15 [error] 1556#0: *503 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:21:42 [error] 1556#0: *505 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:21:45 [error] 1556#0: *505 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:24:24 [error] 1556#0: *510 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:37:46 [error] 1553#0: *515 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:38:08 [error] 1553#0: *515 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
2018/10/01 14:38:54 [error] 1553#0: *515 open() "/opt/yxf_mysite_py_django/mysite/static/images/besttome.gif" failed (2: No such file or directory), client: 192.168.2.1, server: localhost, request: "GET /static/images/besttome.gif HTTP/1.1", host: "192.168.2.135", referrer: "http://192.168.2.135/"
